web/v1/snowflake/sql-runner/sql/standard/00-setup/01-main/01-stored-procedures.sql

```diff
@@ -89,12 +89,18 @@ CREATE OR REPLACE PROCEDURE {{.output_schema}}.column_check(SOURCE_SCHEMA VARCHA
   var delim = '~';
   var sourceColumns = list_cols_with_type(SOURCE_SCHEMA,SOURCE_TABLE,delim).split(delim);
   var targetColumns = list_cols_with_type(TARGET_SCHEMA,TARGET_TABLE,delim).split(delim);
+  var sourceColumnLengths = list_cols_with_length(SOURCE_SCHEMA,SOURCE_TABLE,delim).split(delim);
```

Comment:
Thank you for the valuable feedback, I agree that it could be cleaner. Just added a reworked stored procedure commit, similar to the mobile model, without helper functions.